test/models/submission_test.rb

```diff
@@ -316,6 +316,7 @@ class SubmissionTest < ActiveSupport::TestCase
       author: {
         name: author.name,
         reputation: 50,
+        flair: author&.flair,
```

Comment:
This also applies to the other occurences in this file

```suggestion
        flair: author.flair,
```